### PR TITLE
Localise some notifications

### PR DESCRIPTION
**Pull request by Feodor0090**
Adds localization for the overlay and several notifications which caught my eye. Didn't passed them all because i want to get feedback am i doing everything right first.

### CODE REVIEW

File: osu.Game/Overlays/NotificationOverlay.cs [hunk 1]
```diff
@@ -92,8 +92,8 @@ private void load()
                                     RelativeSizeAxes = Axes.X,
                                     Children = new[]
                                     {
-                                        new NotificationSection(AccountsStrings.NotificationsTitle, new[] { typeof(SimpleNotification) }, "Clear All"),
-                                        new NotificationSection(@"Running Tasks", new[] { typeof(ProgressNotification) }, @"Cancel All"),
+                                        new NotificationSection(AccountsStrings.NotificationsTitle, new[] { typeof(SimpleNotification) }, NotificationsStrings.ClearAll),
```
Comment (Joehuu) [inline osu.Game/Overlays/NotificationOverlay.cs:95]: Can use existing web translation (seen here):
image

```suggestion
                                        new NotificationSection(AccountsStrings.NotificationsTitle, new[] { typeof(SimpleNotification) }, NotificationsStrings.MarkRead(NotificationsStrings.FiltersDefault)),
```

Class names are going to conflict though.

Comment (bdach) [reply osu.Game/Overlays/NotificationOverlay.cs]: I'll be honest, I would be kind of cautious with reusing translations like that. You can't know whether all usages are going to be in the exact same context with that sort of reuse. The necessity of having to auto-fill `FiltersDefault` here is also making me a little worried.

I guess we can go with it until someone reports a problem, though.

File: osu.Game/Localisation/NotificationsStrings.cs [hunk 2]
```diff
@@ -15,10 +15,48 @@ public static class NotificationsStrings
...
+
+        /// <summary>
+        /// "Cancel All"
+        /// </summary>
+        public static LocalisableString CancelAll => new TranslatableString(getKey(@"cancel_all"), @"Cancel All");
+
+        /// <summary>
+        /// "Your battery level is low! Charge your device to prevent interruptions during gameplay."
+        /// </summary>
+        public static LocalisableString LowBatteryWarning =>
```
Comment (bdach) [inline osu.Game/Localisation/NotificationsStrings.cs:35]: nit: would rename to `BatteryLow` (to match `GameVolumeTooLow`)

File: osu.Game/Localisation/NotificationsStrings.cs [hunk 3]
```diff
@@ -15,10 +15,48 @@ public static class NotificationsStrings
...
+        /// <summary>
+        /// "osu! doesn&#39;t seem to be able to play audio correctly.\n\nPlease try changing your audio device to a working setting."
+        /// </summary>
+        public static LocalisableString AudioPlaybackIssue => new TranslatableString(getKey(@"audio_playback_issue"),
+            @"osu! doesn't seem to be able to play audio correctly.\n\nPlease try changing your audio device to a working setting.");
+
+        /// <summary>
+        /// "The score overlay is currently disabled. You can toggle this by pressing {0}."
+        /// </summary>
+        public static LocalisableString TheScoreOverlayIsDisabled(LocalisableString arg0) => new TranslatableString(getKey(@"the_score_overlay_is_disabled"),
```
Comment (bdach) [inline osu.Game/Localisation/NotificationsStrings.cs:57]: nit: would rename to `ScoreOverlayDisabled`

File: osu.Game/Overlays/NotificationOverlay.cs (hunk 1 above)
Comment (Loreos7) [reply osu.Game/Overlays/NotificationOverlay.cs]: Please don't use it from the web, in Russian this has another context.

**Review by bdach — APPROVED** [review]

**Review by Joehuu — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**